hxrsnd/aerotech.py

```diff
@@ -145,7 +145,7 @@ def move(self, position, *args, **kwargs):
             # Check the motor status
             self.check_status()
             
-            return super().move(position, *args, **kwargs)
+            return super().move(position, wait=False, *args, **kwargs)
         except KeyboardInterrupt:
             self.stop()
```

Comment:
Still need the `KeyboardInterrupt` if you are never waiting?

Reply:
Good catch. Removing those lines.